abstract/equations/forward-pass.tex

```diff
@@ -0,0 +1,7 @@
+\vspace{-0.1cm}
```

Comment:
@tetikshajain: would you mind going through the files in `equations` and `figures` and commenting out the vspace lines? There will often be 2, the one at the top and the one right below the label